### PR TITLE
Tickets/dm 8876

### PR DESCRIPTION
**Pull request by pschella**

### CODE REVIEW

File: python/lsst/shapelet/shapeletLib.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+from ._gaussHermiteProjection import *
+from ._gaussHermiteConvolution import *
+from ._multiShapeletFunction import *
+from .multiShapeletFunction import *
+from ._multiShapeletBasis import *
+from ._matrixBuilder import *
+from ._hermiteTransformMatrix import *
+from ._radialProfile import *
+from .radialProfile import *
+from ._functorKeys import *
```
Comment (fred3m) [inline python/lsst/shapelet/shapeletLib.py]: Please add newline

File: python/lsst/shapelet/__init__.py [hunk 2]
```diff
@@ -89,3 +90,4 @@ def __iter__(self):
             if p == q:
                 yield (i, p, q, True)
                 i += 1
+
```
Comment (pschella) [inline python/lsst/shapelet/__init__.py]: I agree, but this was preexisting and there isn't an obvious place for it. How do we flag it?

Comment (fred3m) [inline python/lsst/shapelet/__init__.py]: Should it be added to Dm-9155?

File: python/lsst/shapelet/basisEvaluator.cc [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    clsBasisEvaluator.def("fillEvaluation",
+                          [](BasisEvaluator &self, Array1d const &array, afw::geom::Extent2D const &point) {
+                              return self.fillEvaluation(array, point);
+                          },
+                          "array"_a, "point"_a);
+    clsBasisEvaluator.def(
+        "fillEvaluation",
+        [](BasisEvaluator &self, Array1d const &array, afw::geom::Extent2D const &point, Array1d const &dx,
+           Array1d const &dy) { return self.fillEvaluation(array, point, dx, dy); },
+        "array"_a, "point"_a, "dx"_a, "dy"_a);
```
Comment (pschella) [inline python/lsst/shapelet/basisEvaluator.cc]: ok

File: python/lsst/shapelet/shapeletLib.py (hunk 1 above)
Comment (pschella) [inline python/lsst/shapelet/shapeletLib.py]: ok

File: python/lsst/shapelet/__init__.py (hunk 2 above)
Comment (pschella) [inline python/lsst/shapelet/__init__.py]: I moved it to a new file `generator.py`.

**Pull request merged**